Dockerfile

```diff
@@ -6,9 +6,7 @@ RUN apt-get update \
     && apt-get install -y curl libcurl4-openssl-dev ruby ruby-dev make
 
 # install fluentd with plugins
-RUN gem install fluentd --no-ri --no-rdoc \
-    && fluent-gem install fluent-plugin-elasticsearch \
-    fluent-plugin-record-modifier fluent-plugin-exclude-filter \
+RUN gem install fluentd fluent-plugin-elasticsearch --no-ri --no-rdoc \
```

Comment:
The `fluent-plugin-elasticsearch` plugin should be replaced with `fluent-plugin-cloudwatch-logs`. cc @bromanko